ibis/backends/duckdb/__init__.py

```diff
@@ -988,6 +988,46 @@ def read_postgres(
 
         return self.table(table_name)
 
+    def read_mysql(
+        self,
+        uri: str,
```

Comment:
Can we make all of these arguments required-keyword?

Reply:
Do you mean not, by not allowing table to be None, and trigger the Value Error later? 

Should I remove that from the postgres and sqlite cases too?